sdk/java-sdk-protobuf-testkit/src/main/java/kalix/javasdk/testkit/EventingTestKit.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.javasdk.testkit;
+
+import akka.actor.ActorSystem;
+import akka.annotation.ApiMayChange;
+import akka.annotation.InternalApi;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.GeneratedMessageV3;
+import kalix.javasdk.Metadata;
+import kalix.javasdk.impl.MessageCodec;
+import kalix.javasdk.testkit.impl.EventingTestKitImpl;
+
+import java.time.Duration;
+import java.util.List;
+
+
+public interface EventingTestKit {
+
+  /**
+   * INTERNAL API
+   */
+  @InternalApi
+  static EventingTestKit start(ActorSystem system, String host, int port, MessageCodec codec) {
+    return EventingTestKitImpl.start(system, host, port, codec);
+  }
+
+
+  Topic getTopic(String topic);
+
+  /**
+   * Testkit utility to mock broker's topic. Useful when doing integration tests for services that do eventing (in or out) to a broker's topic.
+   */
+  @ApiMayChange
+  interface Topic {
+
+    /**
+     * Waits for predefined amount of time. If a message arrives in the meantime or
```

Comment:
Maybe some information about the default timeout value, which could be a doc-link to the constant?